pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator.go

```diff
@@ -64,9 +68,17 @@ func (mutator *VMIsMutator) Mutate(ar *admissionv1.AdmissionReview) *admissionv1
 	}
 
 	var patch []utiltypes.PatchOperation
+	var causes []metav1.StatusCause
```

Comment:
It's used in both branches of the following conditional, that's why it's here alongside patch that is likewise used in both.

Reply:
I used "find all usages" functionality in the IDE, and it only found usage on lines 204, 212, 219, 220. That is because line 77 uses `:=` which defines a new variable with the same name for that scope.

I think, if the `causes` variable is not used outside of the `if` branches, it can be defined inside each one as a separate variable.